app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_input = command.split()
+    if splitted_input[1] != splitted_input[2]:
```

Comment:
You should check that command have exactly 3 parameters, command is exactly "cp"